packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -378,7 +337,7 @@ export class TableLayout<T> extends ListLayout<T> {
 
     this.setChildHeights(children, height);
 
-    rect.width = this.layoutInfos.get('header').rect.width;
+    rect.width = this.layoutInfos.get(this.collection.head?.key ?? 'header').rect.width;
     rect.height = height + 1; // +1 for bottom border
```

Comment:
I noticed that our layouts have some extra calculations built in like the above that adds 1 to the row height for the border. Will this be a bit strange for a RAC user who provides something like `rowHeight: 25` to their TableLayout but then gets an actual row height of 26px?

Reply:
hmm yeah. I am wondering if we split out a spectrum-specific layout from the generic one...

Reply:
IMO it would ideal to split them. I'll add to the canvas to discuss this further